lib/actions/plan.js

```diff
@@ -96,17 +109,22 @@ export function planNext () {
  * Plan the last trip of the day, or if the last trip has already been planned,
  * plan the last trip of the next day.
  */
-export function planLast () {
+export function planLast() {
   return function (dispatch, getState) {
     const state = getState()
     const itinerary = getActiveItinerary(state)
-    const {currentQuery} = state.otp
+    const { currentQuery } = state.otp
     const params = {
-      date: moment(currentQuery.date).add('days', 1).format(OTP_API_DATE_FORMAT),
+      date: moment(currentQuery.date)
+        .add('days', 1)
+        .format(OTP_API_DATE_FORMAT),
       departArrive: 'ARRIVE',
-      startTransitStopId: getFirstStopId(itinerary),
       time: SERVICE_BREAK
     }
+    if (!state.otp?.config?.api?.v2) {
+      params.startTransitStopId = getFirstStopId(itinerary)
```

Comment:
This if statement is repeated several times, could it just go into the `getFirstStopId` function?

Reply:
I tried to make this work but the problem is the field can't be set at all and even undefined creates a value!